src/Controller/EnMarche/CitizenProjectController.php

```diff
@@ -199,4 +204,49 @@ public function listActorsAction(CitizenProject $citizenProject, CitizenProjectM
             'actors' => $citizenProjectManager->getCitizenProjectMemberships($citizenProject),
         ]);
     }
+
+    /**
+     * @Route("/{slug}/acteurs/contact", name="app_citizen_project_contact_actors")
+     * @Method("POST")
+     * @Security("is_granted('ADMINISTRATE_CITIZEN_PROJECT', citizenProject)")
```

Comment:
could we move this action in the `CitizenProjectManagerController` ? it has the same security annotations